src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetSerializableV1.java

```diff
@@ -1107,6 +1108,8 @@ else if (getQueryResultFormat().equals(QueryResultFormat.ARROW)) {
           firstChunkRowCount += root.getRowCount();
           root.clear();
         }
+      } catch (ClosedByInterruptException cbie) {
+        logger.debug("Interrupted when loading Arrow first chunk row count.", cbie);
```

Comment:
Can you put some comment, we continue in loop or retry? We just log the execption.